packages.dhall

```diff
@@ -1,4 +1,4 @@
 let upstream =
-      https://raw.githubusercontent.com/purescript/package-sets/prepare-0.14/src/packages.dhall
+      https://github.com/purescript/package-sets/releases/download/psc-0.14.0-20210304/packages.dhall sha256:c88151fe7c05f05290224c9c1ae4a22905060424fb01071b691d3fe2e5bad4ca
```

Comment:
Does it matter if the set is `psc-0.14.0-20210304` or `psc-0.14.0`?

https://github.com/purescript-contrib/purescript-github-actions-toolkit/pull/14#issuecomment-792227946

Reply:
It doesn't matter -- I have a _slight_ preference for 0.14.0, but I'll accept either.

Reply:
Also, my comment on the changelog applies to each of these pull requests. Sorry I didn't see it sooner!